roofit/roofitcore/inc/RooArgList.h

```diff
@@ -26,39 +26,20 @@ class RooArgList : public RooAbsCollection {
   RooArgList(const RooArgSet& set) ;
   explicit RooArgList(const TCollection& tcoll, const char* name="") ;
   explicit RooArgList(const char *name);
-  RooArgList(const RooAbsArg& var1, 
-	     const char *name="");
-  RooArgList(const RooAbsArg& var1, const RooAbsArg& var2, 
-	     const char *name="");
-  RooArgList(const RooAbsArg& var1, const RooAbsArg& var2,
-	     const RooAbsArg& var3, 
-	     const char *name="");
-  RooArgList(const RooAbsArg& var1, const RooAbsArg& var2,
-	     const RooAbsArg& var3, const RooAbsArg& var4, 
-	     const char *name="");
-  RooArgList(const RooAbsArg& var1, const RooAbsArg& var2,
-	     const RooAbsArg& var3, const RooAbsArg& var4, 
-	     const RooAbsArg& var5, 
-	     const char *name="");
-  RooArgList(const RooAbsArg& var1, const RooAbsArg& var2,
-	     const RooAbsArg& var3, const RooAbsArg& var4, 
-	     const RooAbsArg& var5, const RooAbsArg& var6, 
-	     const char *name="");
-  RooArgList(const RooAbsArg& var1, const RooAbsArg& var2,
-	     const RooAbsArg& var3, const RooAbsArg& var4, 
-	     const RooAbsArg& var5, const RooAbsArg& var6, 
-	     const RooAbsArg& var7, 
-	     const char *name="");
-  RooArgList(const RooAbsArg& var1, const RooAbsArg& var2,
-	     const RooAbsArg& var3, const RooAbsArg& var4, 
-	     const RooAbsArg& var5, const RooAbsArg& var6, 
-	     const RooAbsArg& var7, const RooAbsArg& var8, 
-	     const char *name="");
-  RooArgList(const RooAbsArg& var1, const RooAbsArg& var2,
-	     const RooAbsArg& var3, const RooAbsArg& var4, 
-	     const RooAbsArg& var5, const RooAbsArg& var6, 
-	     const RooAbsArg& var7, const RooAbsArg& var8, 
-	     const RooAbsArg& var9, const char *name="");
+  /// Construct a (non-owning) RooArgList from one or more
+  /// RooFit objects.
+  /// \param arg A RooFit object to be put in the set.
+  /// \param varsOrName Arbitrary number of
+  ///   - RooFit objects deriving from RooAbsArg.
+  ///   - A c-string to name the set.
+  template<typename... Arg_t>
+  RooArgList(const RooAbsArg& arg, const Arg_t&... argsOrName)
+  /*NB: Making this a delegating constructor lead to linker errors with MSVC*/ {
```

Comment:
```suggestion
  /*NB: Making this a delegating constructor leads to linker errors with MSVC*/ {
```

Reply:
I meant "lead". When microsoft catches up, the statement might become false.

Reply:
"led"?

Reply:
I meant "led"! That's what I said, didn't I? :sweat_smile: